tests/ballerina-unit-test/src/test/resources/test-src/statements/block/block-stmt-negative.bal

```diff
@@ -95,27 +95,27 @@ function testUnreachableBreak() returns (string){
 }
 
 public type testError record {
-    string message;
-    error? cause;
-    string code;
+    string message = "";
+    error cause = error("", {});
+    string? code = ();
```

Comment:
For records as well, default values are needed only if they are/won't be provided in the record literal.

To make a field optional, use `string code?;`